pyproject.toml

```diff
@@ -61,6 +61,11 @@ acceptance = [
 source = "nodejs"
 path = "labextension/package.json"
 
+[tool.hatch.build]
+artifacts = [
+    "jupyter_server_proxy/labextension",
+]
+
```

Comment:
If we are hard coding the extension name, we do not need to distribute the `labextension` files in the package as we do not need to read `package.json` file anymore